src/common/Header/BackBtn.tsx

```diff
@@ -1,11 +1,17 @@
+import { SetStateAction } from 'react';
 import { IcBackDark } from '../../assets/icon';
 import { useNavigate } from 'react-router-dom';
 
-const BackBtn = () => {
+interface BackBtnProps {
+  step: number;
+  setStep: React.Dispatch<SetStateAction<number>>;
+}
+
+const BackBtn = ({ step, setStep }: BackBtnProps) => {
   const navigate = useNavigate();
 
   const handleClickBack = () => {
-    navigate(-1);
+    step === 0 ? navigate('/') : setStep(step - 1);
```

Comment:
backBtn은 바로 이전으로 돌아가는 기능이니까 step을 props로 받지 않고 prev를 이용해서 사용할수도 있을것 같아용!
```suggestion
    step === 0 ? navigate('/') : setStep((prev) => prev - 1);
```
이런식으로용!